source/manual/content-designer-github-access.html.md

```diff
@@ -0,0 +1,60 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Content designer GitHub access
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+last_reviewed_on: 2018-06-01
+review_in: 3 months
+---
+
+Content designers working on GOV.UK may need GitHub access to edit content
+which is stored as code in an application, for example
+[smart answers][smart-answers-github]. For this we have a dedicated GitHub team
+that content designers can be added to. When added to this team a content
+designer can create a branch within a GOV.UK repository and open a pull
+request, however a developer will be required to merge the request into the
+master branch.
+
+When on 2nd Line you may receive a ticket to add/remove content designers from
+this access.
+
+## Adding someone to the GitHub team
+
+When a content designer joins they will need to be added to our monitoring
+software and then can be added to the GitHub team. These are the steps to
+follow:
+
+1. Edit the [users.yml][] file in [govuk-user-reviewer][] with the following
+   details for each designer: their name, GitHub username and role in
+   GOV.UK - [example][user-example]
+1. Raise this as a pull request
+1. Once PR is merged email gds-github-owners@digital.cabinet-office.gov.uk
```

Comment:
Can we change this to govuk-senior-tech-members@digital.cabinet-office.gov.uk? The GDS GitHub owners mailing list goes out to a lot more people, who won't have context. I wouldn't want to receive emails for access requests in other parts of GDS.

Reply:
@tijmenb 👍  fixed in 212475f